code/modules/mob/living/carbon/human/human_damage.dm

```diff
@@ -31,17 +26,45 @@
 	tod = 0
 	timeofdeath = 0
 
+/mob/living/carbon/human/adjustBrainLoss(var/amount)

```

Comment:
Wait---does this mean brainloss is handled entirely on the organ now and not by the brainloss var orrr?